resfulservice/misc/README.md

```diff
@@ -0,0 +1,9 @@
+1. Download gpg [here](https://gpgtools.org/) and request decryption keys for `envVarFile.gpg`
+
+On the terminal:
+2. Decrypt `envVarFile.gpg` content by executing `decryptENVARFile.sh` in this directory
+3. Go to => `~/.bash_profile` (If not found create one)
+4. Scroll down to the end of the `.bash_profile` file
+5. Copy and paste `envVarFile` content
+6. Save any changes you made to the `.bash_profile` file
+7. Execute the new .bash_profile by running => `source ~/.bash-profile`
```

Comment:
This doesn't appear to render as intended

https://github.com/Duke-MatSci/materialsmine/blob/952115524b3c52516639d28472075b8d09ee4905/resfulservice/misc/README.md

Reply:
😂 . I'll take a look, nice catch